README.md

```diff
@@ -42,15 +42,30 @@ If you run into problems using the SDK, you can:
 * Ask questions on the [Okta Developer Forums][devforum]
 * Post [issues][github-issues] here on GitHub (for code errors)
 
-### Browser Compatibility
+### Browser compatibility / polyfill
 
 This SDK is known to work with current versions of Chrome, Firefox, and Safari on desktop and mobile.
 
-Compatibility with IE Edge can be accomplished by adding polyfill/shims for the following objects:
+Compatibility with IE 11 / Edge can be accomplished by adding polyfill/shims for the following objects:
 
-* ES6 Promise
+* ES Promise
 * Array.from
 * TextEncoder
+* Object.assign
+* UInt8 typed array
+* webcrypto (crypto.subtle)
+
+This module provides an entrypoint that implements all required polyfills. To use it, import or require `@okta/okta-auth-js/polyfill` at or near the beginning of your application's code.
```

Comment:
Add an option to use it in the `script` tag?

Reply:
That's what I was discussing about "the CDN option"

Reply:
You *could* include the webpack bundle from the node modules directory I suppose

Reply:
yep, I remember there are plugins for that.